.readthedocs.yaml

```diff
@@ -0,0 +1,35 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.10"
+  apt_packages:
+    - curl
+    - jq
+  # https://docs.readthedocs.io/en/latest/build-customization.html
+  jobs:
+    pre_build:
+      - python -m sphinx -b linkcheck docs/ _build/linkcheck
```

Comment:
I guess a question is should we bother running the linkcheck builder on RTD if we're also running it in the GitHub Actions workflow?